com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -193,8 +196,15 @@ public new void OnEnable()
             foreach (var cameraTarget in targets)
             {
                 var additionData = (cameraTarget as Component).gameObject.GetComponent<UniversalAdditionalCameraData>();
-                if(additionData == null)
+                if (additionData == null)
+                {
                     additionData = (cameraTarget as Component).gameObject.AddComponent<UniversalAdditionalCameraData>();
+#if ENABLE_VR && ENABLE_XR_MODULE
+                    List<XRDisplaySubsystemDescriptor> displayDescriptors = new List<XRDisplaySubsystemDescriptor>();
+                    SubsystemManager.GetSubsystemDescriptors(displayDescriptors);
+                    additionData.allowXRRendering = displayDescriptors.Count > 0;
```

Comment:
I don't think we should be changing serialized component data in `OnEnable`. We shouldn't have to change this value based on the presence of XR. It should be enough to have the checks during rendering.

Reply:
For the fogbugz case, the runtime check is good enough. 
The code here improved the default value for camera's target eye. It will defaults new camera's target eye field to None when it gets created in editor and no XR provider is installed.

I will revert the change here and put it into another PR. We can review it separately.

Reply:
Addressed in 
1d87eaa